iina/Base.lproj/KeyBinding.strings

```diff
@@ -52,6 +52,7 @@
 "iina.toggle-flip" = "Toggle flip";
 "iina.toggle-mirror" = "Toggle mirror";
 "iina.save-playlist" = "Save current playlist";
+"iina.reveal-current-file" = "Reveal current file in Finder";
```

Comment:
I will try and find time for a proper review tomorrow. As per issue #3976 this must be show, not reveal. I'd prefer just `Show in Finder` as that is the common label for such a menu item. I think it is ok that it differs from the label for deleting. See what the other reviewers say.

Reply:
The way our integration with Crowdin works whenever you change a file in Base.lproj the corresponding file in en.lproj needs to be updated as well. No other language files should be changed. They will be updated through Crowdin.

Reply:
I'd like "Show current file in Finder". "Show in Finder" is kinda weird to me since there's no subject.

Reply:
I don't see a significant difference in popularity
https://github.com/search?q=%22Reveal+in+Finder%22&type=code
https://github.com/search?q=%22Show+in+Finder%22&type=code
In VSCode it's "Reveal", in XCode it's "Show". Change to Show?

Reply:
We had a discussion in #3976 before and the conclusion was to use "show" instead of "reveal" all across IINA. So please change to show to make it consistent with other parts of IINA.

Reply:
Go with @uiryuu "Show current file in Finder" although I prefer "Show in Finder" as that is the typical Apple menu item, like in Xcode. The capitalization needs to be "Show Current File in Finder" as per Apple HIG.

The way I update the en.lproj strings file for a XIB is to use [ibtool](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/MaintaingYourOwnStringsFiles/MaintaingYourOwnStringsFiles.html#//apple_ref/doc/uid/10000171i-CH19-SW6) like so:
```text
ibtool iina/Base.lproj/MainMenu.xib --generate-strings-file temp.strings
```

Then I run [meld](https://github.com/yousseb/meld):
```text
meld temp.strings iina/en.lproj/MainMenu.strings
```

And merge in only the string I added.

Reply:
OK, done. BTW note that the screenshot OSD use "REVEAL" word.

Reply:
done